protx-client/src/components/ProTx/components/charts/AnalyticsStateDistribution.jsx

```diff
@@ -3,6 +3,7 @@ import { useDispatch, useSelector } from 'react-redux';
 import PropTypes from 'prop-types';
 import { LoadingSpinner } from '_common';
 import MainPlot from './MainPlot';
+import styles from './MainChart.module.scss';
```

Comment:
Actually, you could drop this line. I don't think it impacts the class being in the component code because it is available up the chain at render.